examples/illustrations/cell_site/src/cell_site-site.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "@id": "kb:dc9b8413-f681-4bc6-a66e-b70a7ecde4d4",
+        "@type": "drafting:CellSite",
+        "uco-core:hasFacet": [
+            {
+                "@type": "drafting:CellSiteFacet",
+                "drafting:cellSiteType": "GSM",
+                "drafting:cellSiteCountryCode": "228",
+                "drafting:cellSiteNetworkCode": "2",
+                "drafting:cellSiteLocationAreaCode": "22100",
+                "drafting:cellSiteIdentifier": "29220952"
+            },
+            {
+                "@type": "drafting:AntennaFacet",
+                "drafting:azimuth": {
+                    "@type": "xsd:decimal",
+                    "@value": "45"
```

Comment:
45 degrees from what?  Is this True North based Azimuth ( https://en.wikipedia.org/wiki/Azimuth#True_north-based_azimuths )?  If the Wikipedia definition specifies what this property means, we have been OK citing Wikipedia for other properties.  Please review the observable namespace's Turtle file for wikipedia.org citation practices.